docs/src/main/paradox/docs/delta/api/current/kg-resolvers-api.md

```diff
@@ -57,9 +58,9 @@ resource with a particular @id.
 The scope of the resolution is the collections of projects `P` defined on the resolver. CrossProject resolution also 
 defines a identity policy `I` (via the `identities` or the `useCurrentCaller` fields) to enforce ACLs. In other words:
 
-- Schema `A` can import schema `B` using the `owl:import` as long as schema `B` is located in some of the projects from 
+- Schema `A` can import schema `B` using the `owl:imports` as long as schema `B` is located in some of the projects from 
   the collection `P` and as long `I` have `resources/read` permissions on the schema `B` project.
-- Resource `A` can reference resource's context `B` (inside `@context`) as long as resource `B` is located in some of 
+- Resource `A` can reference to a remote context existing in resource `B` as long as resource `B` is located in some of 
```

Comment:
I think a very similar sentence exists for the `CrossProject` resolver section. It would be good to align it.